.gitignore

```diff
@@ -2,3 +2,5 @@
 .project
 .classpath
 .settings/
+.idea/
```

Comment:
There was no requirement to amend .gitignore file, please discard changes.

Reply:
I think this is necessary